protocol-units/da/m1/util/src/config/local/m1_da_light_node.rs

```diff
@@ -1,13 +1,90 @@
 use crate::config::common::{
 	default_celestia_rpc_connection_hostname, default_celestia_rpc_connection_port,
 	default_celestia_websocket_connection_hostname, default_celestia_websocket_connection_port,
-	default_da_signers_sec1_keys, default_da_signing_private_key,
 	default_m1_da_light_node_connection_hostname, default_m1_da_light_node_connection_port,
 	default_m1_da_light_node_listen_hostname, default_m1_da_light_node_listen_port,
 };
+use ecdsa::SigningKey;
+use k256::Secp256k1;
 use serde::{Deserialize, Serialize};
 use std::collections::HashSet;
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct DaSigners {
+	pub da_signing_private_key_hex: String,
+	pub da_signers_public_keys_hex: HashSet<String>,
+}
+
+/// The default da signing private key
+pub fn default_da_signing_private_key() -> SigningKey<Secp256k1> {
+	match std::env::var("DA_SIGNING_PRIVATE_KEY") {
+		Ok(val) => {
+			// decode from hex to bytes 32
+			let hex_bytes = hex::decode(val).expect("Invalid hex string");
+
+			// todo: maybe remove the unwrap and catch for a random signing key
+			let signing_key_bytes: &[u8; 32] =
+				hex_bytes.as_slice().try_into().expect("Slice with incorrect length");
+			SigningKey::from_bytes(signing_key_bytes.into()).unwrap()
+		}
+		Err(_) => SigningKey::random(
```

Comment:
This case could be for the unset value:

```suggestion
		Err(env::VarError::NotPresent) => SigningKey::random(
```

And then the other error case could be panicked on, but that's finessing.